SPECS-EXTENDED/ocaml-calendar/ocaml-calendar.spec

```diff
@@ -1,29 +1,40 @@
 Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
+
+# OCaml packages not built on i686 since OCaml 5 / Fedora 39.
+ExcludeArch: %{ix86}
+
 %ifnarch %{ocaml_native_compiler}
 %global debug_package %{nil}
 %endif
 
+# ocaml-alcotest requires ocaml-astring, ocaml-cmdliner, ocaml-fmt, and ocaml-uutf,
+# none of which are otherwise needed for building the OCaml-dependent packages
+# found in RHEL and ELN.  We want to avoid the extra dependencies there.
+%bcond tests %[!0%{?rhel}]
+
+%global giturl  https://github.com/ocaml-community/calendar
+
+
 Name:           ocaml-calendar
-Version:        2.04
-Release:        29%{?dist}
+Epoch:          1
+Version:        3.0.0
+Release:        1%{?dist}
 Summary:        Objective Caml library for managing dates and times
-License:        LGPLv2
-
-URL:            https://github.com/ocaml-community/calendar
-Source0:        https://download.ocamlcore.org/calendar/calendar/%{version}/calendar-%{version}.tar.gz
+License:        LGPL-2.1-or-later WITH OCaml-LGPL-linking-exception
 
-Patch1:         calendar-2.03.2-enable-debug.patch
+URL:            https://ocaml-community.github.io/calendar/
+VCS:            git:%{giturl}.git
+Source0:        %{giturl}/archive/v%{version}/calendar-%{version}.tar.gz
```

Comment:
Any specific reason why the official URL got updated? Previously it was `download.ocamlcore.org`